Modules/Generators/PackageGenerator/Sources/PackageGenerator/Utils/Shell.swift

```diff
@@ -0,0 +1,51 @@
+import ArgumentParser
+import Foundation
+
+struct Shell {
+    var verbose: Bool = false
+
+    @discardableResult
+    func execute(_ command: String) throws -> String {
+        let process = Process()
+        let pipe = Pipe()
+
+        process.executableURL = URL(fileURLWithPath: "/bin/zsh")
+        process.arguments = ["-c", command]
+        process.standardOutput = pipe
+        process.standardError = pipe
+
+        if verbose {
+            Task {
+                await Console.print(.computer, command)
+            }
+        }
+
+        try process.run()
+        process.waitUntilExit()
+
+        let data = pipe.fileHandleForReading.readDataToEndOfFile()
+        if let output = String(data: data, encoding: .utf8) {
+            if verbose {
+                output.split(separator: "\n").forEach { line in
+                    Task {
+                        await Console.print(level: 1, .none, String(line))
+                    }
+                }
```

Comment:
```suggestion
                Task {
                    output.split(separator: "\n").forEach { line in
                        await Console.print(level: 1, .none, String(line))
                    }
                }
```

Here it's best to create 1 task for all lines instead of 1 task per line.